pkg/cmd/container/create.go

```diff
@@ -123,6 +124,37 @@ func Create(ctx context.Context, client *containerd.Client, args []string, netMa
 	}
 	opts = append(opts, platformOpts...)
 
+	if imageRef := args[0]; strings.HasPrefix(imageRef, "oci-archive://") {
```

Comment:
Protocol prefixes should be parsed in the same place as ipfs://

https://github.com/containerd/nerdctl/blob/f240e58d2e5922642128fee245d933a6550ddb46/pkg/referenceutil/referenceutil.go#L91